microk8s-resources/actions/coredns.yaml

```diff
@@ -32,7 +32,7 @@ data:
           fallthrough in-addr.arpa ip6.arpa
         }
         prometheus :9153
-        forward . 8.8.8.8 8.8.4.4
+        forward . 8.8.8.8 8.8.4.4 /etc/resolv.conf
```

Comment:
You can't add resolvers that produce different results like this. 
Ref: https://coredns.io/plugins/forward/

The default policy for forward is random, you're going to mix resolve.conf with 8.8.8.8 which would be bad. I think you're better off sticking to just resolv.conf and accepting an argument to replace it with other resolvers.

Reply:
Thank you, yes this will change to add an option for user to pass dns else resolv.conf will be picked by default, like @balchua commented https://github.com/ubuntu/microk8s/pull/1625#issuecomment-703910093 suggested